CHANGELOG.md

```diff
@@ -1,11 +1,22 @@
 
 # redisio
 
+This file is used to list changes made in each version of the redisio cookbook.
+
 ## Unreleased
 
+- Set unified_mode true for Chef 17+ support
+- Require Chef 15.3+ for unified_mode
```

Comment:
wording here feels wrong as we are going to 16+ due to user_ulimit

Reply:
I split it out into two lines so I could call out both bumps.